syntax = "proto3";

package legumeinfo.microservices.genesearch_service.v1;


service GeneSearch {
  rpc Search (GeneSearchRequest) returns (GeneSearchReply) {}
}


message GeneSearchRequest {
  string query = 1;
}


message GeneSearchReply {
  repeated string genes = 1;
}
